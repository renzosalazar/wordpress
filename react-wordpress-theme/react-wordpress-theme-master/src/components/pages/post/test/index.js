/**
 * Test for post component.
 *
 * @package
 */

/**
 * External dependencies
 */
import renderer from 'react-test-renderer';

/**
 * Internal dependencies
 */
import Post from '../index';

describe( 'Test Post Component', () => {
	it( 'Should match Post snapshot and render correctly', () => {
		const component = renderer.create( <Post /> );

		const tree = component.toJSON();

		expect( tree ).toMatchSnapshot();
	} );
} );
