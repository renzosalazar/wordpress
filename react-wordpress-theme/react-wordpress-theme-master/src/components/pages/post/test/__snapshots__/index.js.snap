// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Post Component Should match Post snapshot and render correctly 1`] = `"Loading..."`;
